tests/Models/GroupTest.php

```diff
@@ -34,6 +34,15 @@ public function testRenameUnchanged()
         $this->assertSame($this->getGroupRenameEventCount($group), $groupRenameEventCount);
     }
 
+    public function testRefreshCacheOnSave()
+    {
+        $previousCacheVersion = cache()->get('groups_local_cache_version');
+
+        factory(Group::class)->create();
+
+        $this->assertNotSame($previousCacheVersion, cache()->get('groups_local_cache_version'));
```

Comment:
this test doesn't actually work because the refresh cache is done in unrelated process by the factory engine when it loads available factories.

also the aftercommit doesn't run because tests are run in one big transaction so there's never a final commit. The transaction thing is disabled wherever aftercommit is needed in test. It probably should be fixed to account test transaction or something. @notbakaneko should know more details on this (and perhaps add documentation somewhere).

Reply:
i fixed the test but also commented it out since it now always fails (as expected, unless test transaction system changes)